deploy/compliance-operator-chart/crds/compliance.openshift.io_compliancecheckresults_crd.yaml

```diff
@@ -0,0 +1 @@
+../../crds/compliance.openshift.io_compliancecheckresults_crd.yaml
```

Comment:
Wondering about the symlinks too (just because my client screeched at me about it). I was looking at the helm options thinking maybe there's a way to redirect it to look for the crd's. Didn't find anything.

In lieu of symlinking these files, you might add a step to the `make bundle` target that copies the CRDs to the helm spot. That'll keep us up to date with changes when new versions are cut.

Reply:
Updated the Makefile for this, too. Fixed in the most recent version.